doc/changes/changes_0.6.2.md

```diff
@@ -0,0 +1,17 @@
+# udf-debugging-java 0.6.2, released 2022-06-07
+
+Code name: Dependency updates / CVE fixes
+
+## Summary
+
+Dependency updates
```

Comment:
I remember Sebastian asked us to use more specifc names for the release summary. Maybe better:
```suggestion
Dependency updates for test-setup-abstranction and project-keeper
```